src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/DesignTimeTargets/Microsoft.Managed.DesignTime.targets

```diff
@@ -118,6 +118,7 @@
     <!-- Capabilities for WPF and WinForms -->
     <ProjectCapability Include="WPF" Condition="'$(UseWPF)' == 'true'" />
     <ProjectCapability Include="WindowsForms" Condition="'$(UseWindowsForms)' == 'true'" />
+    <ProjectCapability Include="DataSourceWindow" Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework' Or '$(_TargetFrameworkVersionWithoutV)' >= '3.0'"/>
```

Comment:
That will enable it for:

UWP
Portable
Anything over version 3.0.

Reply:
I'm not familiar with how those work. Do they have different `TargetFrameworkIndentifier`s, or is there some other way to detect them?

Or should I just restrict this to `.NETFramework Or (.NETCoreApp And >= 3.0)`?

Reply:
The latter.

Reply:
I think we'll want to start having some structure coordination to these TFMs checks so that's easier to see what's enabled for a given TFM/version and to reduce the amount of conditions that need to be evaluated.

Reply:
Yes, I agree, I wouldn't want to add another one of these. I think @etbyrd needs this same condition for #4355 so converting to a `<Choose>` probably makes sense at that point.

Reply:
Late, but should this also be supported for any .NETStandard TFMs, or does it require stuff that's only in .NETCoreApp?

Reply:
As far as I know the Data Sources Window is only useful as a tool to automatically create controls on a designer surface, so wouldn't make sense in .NET Standard